doc/nat.md

```diff
@@ -1,6 +1,6 @@
 # Natural numbers
 
-The `Nat` type represents the natural numbers, i.e., arbitrary-precision unsigned integers.
+The `Nat` type represents the natural numbers (i.e., arbitrary-precision unsigned integers).
```

Comment:
I think the former is OK as well. See, e.g., https://english.stackexchange.com/a/93658.

Reply:
It would be consistent with the bracket usage on Line 64 (`(e.g., Trepplein and TC)`).